.travis.yml

```diff
@@ -31,7 +31,11 @@ addons:
 
```

Comment:
Currently the pull request is saying that the Travis CI is failing (building the documentation). Since I have access to the full Travis stuff, I'll look into that and see if I can figure out what's going on.